### PR TITLE
Reorganizing the code to take functions from the core

### PR DESCRIPTION
**Pull request by bicatcho11**
## Description

Reorganizing the code to take functions from the core
Display the jobs available within the core

## Checklist

- [ ] I have personally loaded this code into an updated Qbox project and checked all of its functionality.
- [ ] My pull request fits the contribution guidelines & code conventions.

### CODE REVIEW

**Comment by bicatcho11** [comment]
https://github.com/Qbox-project/qbx_core/pull/434

File: client/main.lua [hunk 1]
```diff
@@ -75,9 +76,21 @@ end
 
 local function openEmploymentMenu()
     local jobOptions = {}
-    for job, label in pairsInOrder(sharedConfig.employment.jobs) do
+    local sortedJobs = {}
+
+    for job, data in pairs(JOBS) do
+        if data.enableCityHall then
+            table.insert(sortedJobs, job)
```
Comment (BerkieBb) [inline client/main.lua:83]:
```suggestion
            sortedJobs[#sortedJobs + 1] = job
```

File: client/main.lua [hunk 2]
```diff
@@ -75,9 +76,21 @@ end
...
+        end
+    end
+
+    table.sort(sortedJobs, function(a, b)
+        return JOBS[a].label < JOBS[b].label
+    end)
+
+    for _, job in ipairs(sortedJobs) do
```
Comment (BerkieBb) [inline client/main.lua:91]: Use a numeric for loop instead

**Review by BerkieBb — APPROVED** [review]

**Comment by mafewtm** [comment]
This is commonly used for activity jobs rather than all the jobs. This seems more like a niche personal choice as I don't think many servers would want people to be able to chose just any job from cityhall.

**Comment by mafewtm** [comment]
Closing as we've received no response from the author of this PR as well as this being a nice, personal choice which I don't think should be added at this time.

**Pull request closed without merging**